core/sodasql/common/yaml_helper.py

```diff
@@ -11,3 +11,23 @@ def parse_yaml(yaml_str: str, description: str = None):
             return yaml.load(yaml_str, Loader=yaml.SafeLoader)
         except Exception as e:
             logging.error(f'Parsing YAML failed: {str(e)}: ({description if description else yaml_str})xWW4')
+
+    @staticmethod
+    def validate_numeric_value(column_name, key, value):
+        if value is None:
+            logging.info(f'There is no {key} key of this column name {column_name} to be validated')
```

Comment:
Small style change here:
I'd write it as : `There is no value specified for {key} for column {column_name}`